[2024-05-26T12:54:31.559+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-26T12:54:31.581+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_to_minio.download_kaggle_data manual__2024-05-26T12:54:30.287707+00:00 [queued]>
[2024-05-26T12:54:31.588+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_to_minio.download_kaggle_data manual__2024-05-26T12:54:30.287707+00:00 [queued]>
[2024-05-26T12:54:31.589+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-26T12:54:31.599+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_kaggle_data> on 2024-05-26 12:54:30.287707+00:00
[2024-05-26T12:54:31.606+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=87) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-26T12:54:31.607+0000] {standard_task_runner.py:63} INFO - Started process 118 to run task
[2024-05-26T12:54:31.608+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'kaggle_to_minio', 'download_kaggle_data', 'manual__2024-05-26T12:54:30.287707+00:00', '--job-id', '164', '--raw', '--subdir', 'DAGS_FOLDER/import_data_dag.py', '--cfg-path', '/tmp/tmpzhagh973']
[2024-05-26T12:54:31.609+0000] {standard_task_runner.py:91} INFO - Job 164: Subtask download_kaggle_data
[2024-05-26T12:54:31.646+0000] {task_command.py:426} INFO - Running <TaskInstance: kaggle_to_minio.download_kaggle_data manual__2024-05-26T12:54:30.287707+00:00 [running]> on host a2da4a24b74b
[2024-05-26T12:54:31.709+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_to_minio' AIRFLOW_CTX_TASK_ID='download_kaggle_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-26T12:54:30.287707+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-26T12:54:30.287707+00:00'
[2024-05-26T12:54:31.710+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-26T12:54:33.454+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-26T12:54:33.456+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-26T12:54:33.463+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=kaggle_to_minio, task_id=download_kaggle_data, run_id=manual__2024-05-26T12:54:30.287707+00:00, execution_date=20240526T125430, start_date=20240526T125431, end_date=20240526T125433
[2024-05-26T12:54:33.510+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-26T12:54:33.529+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-26T12:54:33.532+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
